config/develop/glue-workflows.yaml

```diff
@@ -0,0 +1,10 @@
+template_path: glue-workflows.yaml
+stack_name: glue-workflows
+parameters:
+  SourceBucketName: !stack_output_external bridge-downstream-dev-source-bucket::BucketName
+  SourceKey: ''
```

Comment:
Blank source key?

Reply:
For now, still have to determine what this is going to be.